adafruit_tfmini.py

```diff
@@ -46,7 +52,7 @@ class TFmini:
     :param timeout: how long we'll wait for valid data or response, in seconds. Default is 1
     """
 
-    def __init__(self, uart, *, timeout=1):
+    def __init__(self, uart: UART, *, timeout: int = 1) -> None:
```

Comment:
Since `timeout` gets used with `time.monotonic()` comparisons, we can type it as `float`.